frontend/src/Context/PublisherProvider/usePublisher/usePublisher.tsx

```diff
@@ -32,7 +33,7 @@ export type AccessDeniedEvent = Event<'accessDenied', Publisher> & {
 };
 
 export type PublisherContextType = {
-  initializeLocalPublisher: () => void;
+  initializeLocalPublisher: (options?: PublisherProperties) => void;
```

Comment:
Hmm, I understand that publisherProperties from the hook can be null while they're 'loading', but this isn't an optional parameter is it? 
In fact you do an early return if options isn't defined where this is invoked